src/nvim/ex_cmds_defs.h

```diff
@@ -176,6 +176,7 @@ typedef struct {
   int lockmarks;                        /* TRUE when ":lockmarks" was used */
   int keeppatterns;                     /* TRUE when ":keeppatterns" was used */
```

Comment:
> I've made everything that is only true or false a bool.

I'd suggest doing that as a separate commit (or even separate PR), since you'll also need to see how the values are used in the rest of the code.  It's not necessarily safe to just blindly change the type.